pkg/clients/grpc/connect.go

```diff
@@ -29,12 +37,21 @@ var (
 	errGrpcClose   = errors.New("failed to close grpc connection")
 )
 
+var (
+	customSEVSNPExtensionOID = asn1.ObjectIdentifier{1, 2, 3, 4, 5, 6}
+	computation              = agent.Computation{}
```

Comment:
on this I think you just need a struct with SNPPolicy and RootOFTrust. So rather than modifying agent.Computation create a new struct with the required fields

Reply:
customSEVSNPExtensionID defines the extension ID, so it's OK to keep it here (ideally it would be constant). As for Computation that's used here only to load policy, please create a new struct.

Reply:
Ok, I will change this. Thanks.

Reply:
Changed.